### PR TITLE
Updated documentation to include summary and second use cas

### PR DESCRIPTION
**Pull request by sheavalentine-mdsol**
This is just a documentation change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # CrichtonTestService
 
--TODO: Write a gem description
+Crichton Test Service is a gem intended to show basic API functionality for a Rails project using the [Crichton](https://github.com/mdsol/crichton) Gem as its API dispatcher.  It allows for testing Hypermedia Clients, Hypermedia Formats, and Ad-Hoc testing.
```
Comment (HonoreDB) [inline README.md]: Last sentence is odd. Suggest something like "It helps support automated and ad-hoc testing of hypermedia-related gems."

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+
+## API Objects
+The API contains a resource called "DRDs" which is a collection of "DRD" objects. DRD stands for Diagnostic and Repair Drone.  These objects support basic CRUD operations.  Included is "leviathon" links, which represents linking to outside resources.  Currently these are fake links, however a future iteration will spin up a second service which serves these resources as well.
```
Comment (HonoreDB) [inline README.md]: s/leviathon/leviathan

File: README.md [hunk 3]
```diff
@@ -18,21 +21,26 @@ Or install it yourself as:
 
 ## Usage
 
+### Automated Hypermedia Testing
 A common usage pattern for this gem will be including code like the following in your spec helper:
 ```ruby
+require 'crichton_test_service'
+
+RAILS_PORT = 8080 # Set the Port that rails to talk to
```
Comment (HonoreDB) [inline README.md]: `# Set the port at which the service can be accessed`

**Comment by sheavalentine-mdsol** [comment]
Is this ready to merge?

**Comment by csavage-mdsol** [comment]
LGTM.  Merging.

**Pull request merged**